doc/api/esm.md

```diff
@@ -1094,16 +1094,23 @@ export async function resolve(specifier, context, defaultResolve) {
 > signature may change. Do not rely on the API described below.
 
 The `getFormat` hook provides a way to define a custom method of determining how
-a URL should be interpreted. This can be one of the following:
+a URL should be interpreted. The `format` returned also affects what the
+acceptable forms of source values are for a module when parsing. This can be one
+of the following:
 
-| `format` | Description |
+| `format` | Description | Acceptable Source Values |
```

Comment:
Maybe it's just me, but I read this as "the source value of `format`", e.g. that `'builtin'` should be a string. I think what you mean is the acceptable type of `source` returned by `getSource` for this format? So maybe call this "Acceptable Types for `getSource` Returned `source`"?

Should we also update the signature in `getSource`?

```js
@returns {string|buffer} response.source
```

Reply:
Yes, I meant the value from either getSource OR transformSource

Reply:
Okay, do you mind making that clearer? Maybe “Acceptable Types For `source` Returned by `getSource` or `transformSource`”?

Reply:
done, also made sure to actually check both hooks rather than just transformSource